spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/Spark3Util.java

```diff
@@ -538,7 +539,7 @@ public static String describe(org.apache.iceberg.SortOrder order) {
   }
 
   public static boolean extensionsEnabled(SparkSession spark) {
-    String extensions = spark.conf().get("spark.sql.extensions", "");
+    String extensions = spark.conf().get(StaticSQLConf.SPARK_SESSION_EXTENSIONS().key(), "");
```

Comment:
let's leave this change out, because there are more places across the codebase that use the hardcoded string which haven't been updated